### ISSUE TITLE
Screen is small on pc when rotated (landscape)

### ISSUE THREAD

**Issue by Karma1233** [Genymobile/scrcpy#481]
This is my issue, detailed in a video so it is clear as to what I'm referring to...
https://youtu.be/l-uwNVsHVDA

I LOVE THIS PROGRAM. I'm not bashing or anything. This thing rocks. Just... would be better if actually went fullscreen or something. It looks like crap and is tiny when in landscape mode. If this could be corrected, it would literally be the BEST mirroring program ever created (at least for how I'm using it).

**Comment by rom1v** [comment]
> This is my issue, detailed in a video so it is clear as to what I'm referring to...

Great report! Thank you.

> I'm not bashing or anything.

Bug reports are a good thing :wink:

On your video, when you rotate the device, a `New texture` should have been printed:

    INFO: Initial texture: 1080x2160
    INFO: New texture: 2160x1080

Do you use the last version (v1.8)? (it was not correctly printed on previous versions)

If it isn't printed with v1.8, then the rotation is not detected on the device-side (in other words, [this callback](https://github.com/Genymobile/scrcpy/blob/v1.8/server/src/main/java/com/genymobile/scrcpy/Device.java#L27) is not called as expected). I think this is the case, because otherwise the window would have been automatically resized, and we still see the not-so-black portrait rectangle.

Does it happen with all apps, or only this camera app (some camera apps have a specific rotation behavior, cf https://github.com/Genymobile/scrcpy/issues/218)?

Might also be related to: https://github.com/Genymobile/scrcpy/issues/186.

**Comment by Karma1233** [comment]
> Do you use the last version (v1.8)? (it was not correctly printed on previous versions)
Yes, I am using v1.8 (had to check and make sure)

> Does it happen with all apps, or only this camera app (some camera apps have a specific rotation behavior, cf #218)?
Turns out, your hunch was correct! It works with other apps that go landscape on my phone (ie Hearthstone, Gmail, Twitch). I guess it's the build it Camera app that has the rotation behavior you're referring to.

To work around this issue with my Camera app, I downloaded a different one (Open Camera) and it works fine. Thanks again for your help. I use this to watch my infant in the other room on my second monitor so that I can get work done at home. 

Just for some extra info, I'm using an LG Stylo 4+ with Android 8.1.0. My Camera App is the Default Camera App that comes with the phone named "Camera" version 7.0.172
Thanks again! Loving Scrcpy!
